apps/framework-cli/src/cli/routines.rs

```diff
@@ -375,7 +449,7 @@ pub async fn start_development_mode(
             api_changes_channel,
             metrics.clone(),
             &mut client,
-            &redis_client,
+            &*redis_client.lock().await,
```

Comment:
and here?

Reply:
yes same as above - one for dev start and one for prod start.